docs/authentication/social-connections/hubspot.mdx

```diff
@@ -29,8 +29,8 @@ Once your app is created, click on the **Auth** tab and copy the **App Id** and
 
 ![Configuring a HubSpot app](/docs/images/authentication-providers/hubspot/02d8bf249b4a6debe85e66ea53b5f88817acdba0-1489x995.png)
 
-In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable HubSpot. In the modal that opened, ensure **Use custom credentials** is enabled and paste the values you obtained during the previous step.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **HubSpot**. Toggle on **Use custom credentials** and paste the values you obtained during the previous step.
 
 Before you close the modal, copy the **Authorized redirect URI**. Go back to the HubSpot panel and paste it into the **Redirect URL** field and click **Save**.
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with HubSpot is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with HubSpot is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with HubSpot is now configured for your instance.
```